medusa/clients/torrent/utorrent_client.py

```diff
@@ -75,17 +113,31 @@ def _add_torrent_file(self, result):
         )
 
     def _set_torrent_label(self, result):
+        torrent_new_label = result.series.name
+
         if result.series.is_anime and app.TORRENT_LABEL_ANIME:
             label = app.TORRENT_LABEL_ANIME
         else:
             label = app.TORRENT_LABEL
 
-        return self._request(params={
-            'action': 'setprops',
-            'hash': result.hash,
-            's': 'label',
-            'v': label,
-        })
+        log.info('torrent label was {path}', {'path': label})
+
+        label = label.replace('%N', torrent_new_label)
+
+        log.info('torrent label is now set to {path}', {'path': label})
+
+        #  Always use show name as label? TBD.
```

Comment:
@p0psicles ?

Reply:
This is commented out ( for review), if  it wouldn't be best to always add the show-name as a label even when the user has no label  defined at all.

Reply:
I don't see any reason to force the use of show name as a label or a label at all.

Reply:
OK, removing...